lib/internal/errors.js

```diff
@@ -1125,6 +1125,56 @@ E('ERR_OUT_OF_RANGE',
     msg += ` It must be ${range}. Received ${received}`;
     return msg;
   }, RangeError);
+E('ERR_QUICCLIENTSESSION_FAILED',
+  'Failed to create a new QuicClientSession: %s', Error);
+E('ERR_QUICCLIENTSESSION_FAILED_SETSOCKET',
```

Comment:
`ERR_QUICCLIENTSESSION_FAILED_SETSOCKET` -> `ERR_QUIC_CLIENT_SESSION_FAILED_SETSOCKET` is easier to read.